packages/relay/src/utils.ts

```diff
@@ -55,4 +58,55 @@ export class Utils {
         throw new Error(`Invalid OPERATOR_KEY_FORMAT provided: ${process.env.OPERATOR_KEY_FORMAT}`);
     }
   }
+
+  /**
+   * Generate logs bloom for synthetic transaction
+   * @param address
+   * @param topics
+   */
+  public static createSyntheticLogsBloom(address: string, topics: string[]): string {
+    if (!topics.length) {
+      return EthImpl.emptyBloom;
+    }
+
+    const items = [address, ...topics];
+    const BYTE_SIZE = 256;
+    const MASK = 0x7ff;
```

Comment:
nit: We can extract these in static constants inside the `Utils` class and reuse them here and inside `checkInLogsBloom`:
```
private static readonly LOGS_BLOOM_BYTE_SIZE = 256;
private static readonly LOGS_BLOOM_MASK = 0x7ff;
```